providers/okta/provider/provider.go

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package provider
+
+import (
+	"errors"
+	"os"
+	"strconv"
+
+	"go.mondoo.com/cnquery/llx"
+	"go.mondoo.com/cnquery/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/providers-sdk/v1/plugin"
+	"go.mondoo.com/cnquery/providers-sdk/v1/upstream"
+	"go.mondoo.com/cnquery/providers-sdk/v1/vault"
+	"go.mondoo.com/cnquery/providers/okta/connection"
+	"go.mondoo.com/cnquery/providers/okta/resources"
+)
+
+type Service struct {
+	runtimes         map[uint32]*plugin.Runtime
+	lastConnectionID uint32
+}
+
+func Init() *Service {
+	return &Service{
+		runtimes:         map[uint32]*plugin.Runtime{},
+		lastConnectionID: 0,
+	}
+}
+
+func (s *Service) ParseCLI(req *plugin.ParseCLIReq) (*plugin.ParseCLIRes, error) {
+	flags := req.Flags
+	if flags == nil {
+		flags = map[string]*llx.Primitive{}
+	}
+
+	conn := &inventory.Config{
+		Type:    req.Connector,
+		Options: make(map[string]string),
+	}
+
+	// flag parsing
+	token := ""
+	if x, ok := flags["token"]; ok && len(x.Value) != 0 {
+		token = string(x.Value)
+	}
+	if token == "" {
+		token = os.Getenv("OKTA_TOKEN")
```

Comment:
This env var is different from the one mentioned at https://github.com/mondoohq/cnquery/pull/1554/files#diff-f41f27f2f43d22fdc7bd16377e3897c34df572efddc892df7ec7d59373ff8c63R59

Reply:
Good catch! fixing